analysis/plotting.py

```diff
@@ -61,14 +61,21 @@ def _formatted_title(benchmark_snapshot_df):
 
 class Plotter:
     """Plotter that uses the same color for the same fuzzer."""
+    # Tableau 20 colors.
+    COLOR_PALETTE = [
+        '#1f77b4', '#aec7e8', '#ff7f0e', '#ffbb78', '#2ca02c', '#98df8a',
+        '#d62728', '#ff9896', '#9467bd', '#c5b0d5', '#8c564b', '#c49c94',
+        '#e377c2', '#f7b6d2', '#7f7f7f', '#c7c7c7', '#bcbd22', '#dbdb8d',
+        '#17becf', '#9edae5'
+    ]
 
     def __init__(self, fuzzers, quick):
         """Instantiates plotter with list of |fuzzers|. If |quick| is True,
         creates plots faster but, with less detail.
         """
-        palette = sns.color_palette(n_colors=len(fuzzers))
         self._fuzzer_colors = {
-            fuzzer: palette[idx] for (idx, fuzzer) in enumerate(sorted(fuzzers))
+            fuzzer: self.COLOR_PALETTE[idx % len(self.COLOR_PALETTE)]
+            for (idx, fuzzer) in enumerate(sorted(fuzzers))
```

Comment:
Can we eliminate the paren around `idx, fuzzer`? Seems nonstandard.

Reply:
Done!